workers/unity/Packages/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
-    "com.unity.ide.rider": "1.1.4",
+    "com.unity.ide.rider": "1.2.1",
```

Comment:
Can we upgrade this to latest while we're at it?

Reply:
Oh fun the package manager doesn't show major updates. (This currently shows as up-to-date, but there is a 2.x version 😕)